src/cai_lab_to_nwb/zaki_2024/behaviorinterface.py

```diff
@@ -33,6 +37,20 @@ def add_to_nwbfile(self, nwbfile: NWBFile, metadata: Optional[dict] = None):
 
         freezing_behavior_df = pd.read_csv(self.file_path)
 
+        #Extract motion data
+        motion_data = freezing_behavior_df["Motion"].values
+
+        motion_series = TimeSeries(
+            name="MotionSeries",
+            description="ezTrack measures the motion of the animal by assessing the number of pixels of the behavioral "
+                        "video whose grayscale change exceeds a particular threshold from one frame to the next.",
+            data=motion_data,
+            unit="n.a",
```

Comment:
We probably can get that from the method but maybe not worth it,  should be something like pixel divided by time.

Reply:
Following best practice it should be S.I. unit, I am not sure there is one for this case

Reply:
Good point. "a.u." atribtrary unit could work as well I guess.